### DISCUSSION TITLE
Please help - Subject: Music

### DISCUSSION THREAD

**Discussion by PolyLittle** [keiwando/toonsquid#30] [Q&A] [answered]
Hello everyone :)
For some reason the my imported audio can't be played while working on project (file format is .wav, the layer is not muted and volume is up). After exporting the project into a movie the sound can be heared but unfortunatly not while working on the project.

I also opened the toonsquid demoproject - same problem there as well. I looked up in the handbook but couldn't find a solution.
Any ideas what I'm doing wrong?  Greetings

**Comment by keiwando** [comment, marked as answer]
Hi! Can you please check to make sure that your iPad is not in Silent Mode (the bell button in Control Center)? Some apps can play audio while the iPad is in Silent Mode and the upcoming 1.6 update will also enable this for ToonSquid.

**Reply by LoneStarz** [reply]
When is the update coming out?

**Reply by PolyLittle** [reply]
WOW :D I actually did not see that. You were right, thanks a lot!
